code_generator/GdkCodeGenerator/src/CodeGeneratorOptions.cs

```diff
@@ -34,6 +35,10 @@ public static CodeGeneratorOptions ParseArguments(ICollection<string> args)
                     "REQUIRED: the directory to output the network types and serialization code to",
                     u => options.NetworkTypesOutputDirectory = u
                 },
+                {
+                    "schema-compiler-path=", "REQUIRED: the schema compiler executable to use",
```

Comment:
This is required, but not checked in the command line validation.

Reply:
Created new PR for these fixes sorry!

https://github.com/spatialos/UnityGDK/pull/113